### DISCUSSION TITLE
how to use http on hostname_replace rather than https

### DISCUSSION THREAD

**Discussion by Sarthac** [searxng/searxng#3127] [Q&A] [answered]
(I don't know should I considered as a bug or not)
I am hosting Searxng on darkweb, using hostname_replace plugin I want to redirect some sites to darknet privacy front-end instances. but by default it adds https rather then http. what should i do to change http over https

**Comment by unixfox** [comment]
As the name implies, it can only replace the hostname not the protocol.

So you will have to modify the protocol either in the source code of the engine or in the settings.yml if the engine allows it.

**Reply by Sarthac** [reply]
okay tell me in short,how do I change the base url or something else
and what do you mean 'in the settings.yml if the engine allows it'

btw I also planning to host i2p instance so this discussion is very important for me

**Comment by return42** [comment, marked as answer]
@osarthak there is no official solution / question has already been asked, see my answer at https://github.com/searxng/searxng/discussions/1294
